no-computer.md

```diff
@@ -11,41 +11,81 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by my idea is the need to play a large range of musical notes which
+can be composed together to make music. I am helping pretty much anyone who would
+want to be able to easily create music and chain notes together to create more complex
+creations. Right now it is difficult to use the entire musical range. Hopefully with
+my new DSL users would be able to chain together unique notes with just their finger
+and have access to a large pitch range for musical compositions.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because to truly compose and play wonderful music one must be
+able to effectively chain together multiple notes and be able to build off the ideas
+of previous compositions. I think the best way to acheive those goals is through a DSL
+since they can allow for a fluency that promotes note chaining and can use previously
+made "works" easily in the program.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+What really excites me about this idea is using the DSL to not only play music but to
+compose new music for others to play much easier than I could before.
+I thought of this because I was just writing about a DSL that would help make beat
+creation simpler and realised there needs to be a good way to play more than just
+beats for those who are unable to use a computer. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Complete Musical Creation and Composition. 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would interact with the language solely with touch. By pressing on
+"buttons" in the language, notes will be played and multiple buttons can be pressed
```

Comment:
The idea of "buttons" is really cool because then it can be anywhere from a child's toy (granted, a rather annoying child's toy) to a composition tool as you're noting. I'm a little confused as to how many "buttons" you are envisioning, if you're trying to use the entire musical range.